### ISSUE TITLE
Invalid registration attempt

### ISSUE THREAD

**Issue by bookerthegeek** [Darkhax-Minecraft/Dark-Utilities#91]
# Heyo #
Saw this in the console and thought you should know.
```
[08:42:39] [Client thread/WARN] [FML]: ****************************************
[08:42:39] [Client thread/WARN] [FML]: * Invalid registration attempt for an Ore Dictionary item with name blockPearl has occurred. The registration has been denied to prevent crashes. The mod responsible for the registration needs to correct this.
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:601)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:587)
[08:42:39] [Client thread/WARN] [FML]: *  at net.darkhax.darkutils.features.shulkerpearl.FeatureShulkerPearlItem.onRegistry(FeatureShulkerPearlItem.java:44)
[08:42:39] [Client thread/WARN] [FML]: *  at net.darkhax.darkutils.DarkUtils.registerBlocks(DarkUtils.java:121)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_20_DarkUtils_registerBlocks_Register.invoke(.dynamic)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)...
[08:42:39] [Client thread/WARN] [FML]: ****************************************
[08:42:39] [Client thread/WARN] [FML]: ****************************************
[08:42:39] [Client thread/WARN] [FML]: * A broken ore dictionary registration with name gemPearl has occurred. It adds an item (type: class net.darkhax.darkutils.features.shulkerpearl.ItemShulkerPearl) which is currently unknown to the game registry. This dictionary item can only support a single value when registered with ores like this, and NO I am not going to turn this spam off. Just register your ore dictionary entries after the GameRegistry.
TO USERS: YES this is a BUG in the mod Dark Utilities report it to them!
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:615)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:587)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:585)
[08:42:39] [Client thread/WARN] [FML]: *  at net.darkhax.darkutils.features.shulkerpearl.FeatureShulkerPearlItem.onRegistry(FeatureShulkerPearlItem.java:45)
[08:42:39] [Client thread/WARN] [FML]: *  at net.darkhax.darkutils.DarkUtils.registerBlocks(DarkUtils.java:121)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_20_DarkUtils_registerBlocks_Register.invoke(.dynamic)...
[08:42:39] [Client thread/WARN] [FML]: ****************************************
[08:42:39] [Client thread/WARN] [FML]: ****************************************
[08:42:39] [Client thread/WARN] [FML]: * Invalid registration attempt for an Ore Dictionary item with name blockWither has occurred. The registration has been denied to prevent crashes. The mod responsible for the registration needs to correct this.
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOreImpl(OreDictionary.java:601)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.oredict.OreDictionary.registerOre(OreDictionary.java:587)
[08:42:39] [Client thread/WARN] [FML]: *  at net.darkhax.darkutils.features.material.FeatureMaterial.onRegistry(FeatureMaterial.java:58)
[08:42:39] [Client thread/WARN] [FML]: *  at net.darkhax.darkutils.DarkUtils.registerBlocks(DarkUtils.java:121)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_20_DarkUtils_registerBlocks_Register.invoke(.dynamic)
[08:42:39] [Client thread/WARN] [FML]: *  at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)...
[08:42:39] [Client thread/WARN] [FML]: ****************************************
```
---
### Versions ###
- DarkUtilities-1.11.2-1.2.0.147
- Bookshelf-1.11.2-2.0.0.388

**Comment by darkhax** [comment]
Thank you for reporting. Looks like the enforced load order from 1.12 has trickled back to 1.11. I don't currently have a good system for updating older versions once I move on so I tend to do them all in one go, when a critical bug is found. I will fix this issue on 1.11 when that happens. This has been fixed on 1.12.

**Comment by bookerthegeek** [comment]
Your welcome.

Enjoy your mod, looking forward to playing on it going forward.
